crates/starknet-proxy-client/src/lib.rs

```diff
@@ -0,0 +1,2 @@
+pub mod proxy_support;
```

Comment:
Please also update the makefile
https://github.com/keep-starknet-strange/zaun/blob/main/Makefile

Reply:
updated